opencti-platform/opencti-front/lang/fr.json

```diff
@@ -2519,8 +2516,35 @@
+  "Unless specific groups are selected, user will be created with default groups.": "Si aucun groupe spécifique n'est sélectionné, l'utilisateur sera créé avec les groupes par défaut.",
+  "You can enable/disable default values for marking in the customization of each entity type.": "Vous pouvez activer/désactiver les valeurs par défaut pour le marquage dans la personnalisation de chaque type d'entité.",
+  "The user's max confidence level overrides the max confidence that might be set at groups level.": "Le niveau de confiance maximal défini par l'utilisateur prévaut sur le niveau de confiance maximal éventuellement défini au niveau des groupes.",
+  "No confidence level found in this user's groups, and no confidence level defined at the user level.": "Aucun niveau de confiance n'a été trouvé dans les groupes de cet utilisateur, et aucun niveau de confiance n'a été défini au niveau de l'utilisateur.",
+  "This group does not have a max confidence level, members might not be able to create data.": "Ce groupe n'a pas de niveau de confiance maximum, les membres pourraient ne pas être en mesure de créer des données.",
+  "This user has no effective confidence level from the groups assigned.": "Cet utilisateur n'a pas de niveau de confiance effectif dans les groupes qui lui ont été attribués.",
+  "confidence_level_from_group": "Cette valeur est héritée du groupe {link}.",
+  "Effective max confidence level:": "Niveau de confiance maximum effectif :",
+  "This value is defined at the user level, which overrides the value inherited from user's groups.": "Cette valeur est définie au niveau de l'utilisateur et remplace la valeur héritée des groupes de l'utilisateur.",
+  "This user is not a member of any group and does not inherit a max confidence level.": "Cet utilisateur n'est membre d'aucun groupe et n'hérite pas d'un niveau de confiance maximal.",
+  "This user does not inherit a max confidence level from their group. Configure user's groups with a max confidence level.": "Cet utilisateur n'hérite pas d'un niveau de confiance maximum de son groupe. Configurer les groupes d'utilisateurs avec un niveau de confiance maximum.",
+  "This group has no max confidence level defined.": "Ce groupe n'a pas de niveau de confiance maximum défini.",
+  "relationship_object-participant": "a participant",
```

Comment:
question: I don't see this in the "en" file, is it normal ?

Reply:
it's in the file. maybe not at the same line, as files other than en are generated.